charts/fleet/templates/job_cleanup_clusterregistrtions.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: fleet-cleanup-clusterregistrations
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded, before-hook-creation
+spec:
+  template:
+    metadata:
+      labels:
+        app: fleet-job
+    spec:
+      serviceAccountName: fleet-controller
+      restartPolicy: Never
+      containers:
+      - name: cleanup
+        image: "{{ template "system_default_registry" . }}{{.Values.agentImage.repository}}:{{.Values.agentImage.tag}}"
+        imagePullPolicy: {{ .Values.global.imagePullPolicy }}
```

Comment:
This is actually empty by default, but we do the same in the [fleet-agent job](https://github.com/rancher/fleet/blob/master/charts/fleet-agent/templates/patch_default_serviceaccount.yaml).